src/main/java/org/folio/circulation/domain/RequestServiceUtility.java

```diff
@@ -32,6 +32,22 @@ private RequestServiceUtility() { }
     }
   }
 
+  static Result<RequestAndRelatedRecords> refuseWhenPagedTlrInstanceDoesNotHaveAvailableItems(
+    RequestAndRelatedRecords requestAndRelatedRecords) {
+
+    Request request = requestAndRelatedRecords.getRequest();
+    if (request.getTlrSettingsConfiguration().isTitleLevelRequestsFeatureEnabled()
+      && request.getRequestType() == RequestType.PAGE
+      && request.getItem().getItem() == null) {
```

Comment:
There should be a method isFound() that does the same

Reply:
Done